### DISCUSSION TITLE
Questions about"jump"(no problem now)

### DISCUSSION THREAD

**Discussion by YaYaYaRong** [dialogic-godot/dialogic#2275] [Q&A] [answered]
I'm very sorry to trouble all of you,but maybe there are some problems about "jump":I can't let it jump to a label but timelines,and its speed is really fast......I'm not sure if it's my problem that I don't really understand the using of jump.

**Comment by YaYaYaRong** [comment, marked as answer]
sorry,I just find that it's because of auto-skip......but could you please tell me where I can find the code?

**Reply by YaYaYaRong** [reply]
I'm really a dumb......sorry

**Reply by CakeVR** [reply]
Has your issue now been fixed? 

Read more about Auto-Advance: https://docs.dialogic.pro/auto-advance.html
